README.md

```diff
@@ -128,6 +128,8 @@ pip3 install --user termtosvg
 choco install neovim
 # -- or --
 scoop install curl
+# -- or --
+nix-env -iA nixpkgs.nixfmt
```

Comment:
You should also add a section [here](https://github.com/tversteeg/emplace/blob/53f8e46d9367990e2d908eab22b53d33f1f98c06/README.md#features) under "Package managers supported:"

Reply:
Good catch, thanks!

Reply:
Fixed it for you :)